src/mjolnir/landmark_builder.cc

```diff
@@ -168,5 +221,44 @@ std::vector<Landmark> LandmarkDatabase::get_landmarks_in_bounding_box(const doub
   return landmarks;
 }
 
+bool BuildLandmarkFromPBF(const boost::property_tree::ptree& pt,
+                          const std::vector<std::string>& input_files) {
+  // parse config to get landmark database
+  auto db = pt.get_optional<std::string>("landmarks");
+  if (!db) {
+    LOG_ERROR("Landmarks config info not found. Landmarks builder will not be created.");
+    return false;
+  }
```

Comment:
its not bad to have this but its not needed exactly.. the config generator will always put a value here, one option would be to do:

```suggestion
  auto db = pt.get<std::string>("landmarks", "");
```

then when we got to make the db it should fail with an invalid path

Reply:
much more concise